### ISSUE TITLE
How to control the volume ratio of different types of particles?

### ISSUE THREAD

**Issue by chenxilong1** [MosGeo/ParticlePack#16]
hi！
I see ResizeType is ByVolume, so I set Bed 1 Grain 1 Proportion = 0.4，Bed 1 Grain 2 Proportion = 0.6， But this can only control the proportion of particle quantity, but cannot control the volume proportion
chen
![2](https://github.com/MosGeo/ParticlePack/assets/115452295/d88c319f-56fa-42fa-b36b-dd18ffcd2584)

**Comment by MosGeo** [comment]
You are correct. The proportion is currently by the number of grains/particles. It is not by volume. The resizetype is unrealted.

 I will keep this in mind for future updates but at this point, I am considering a whole rewrite which will take time.

in the meantime, I think you can write a script to convert your count proportion to volumetric proportion given your pdfs.
